less/index.less

```diff
@@ -54,6 +54,109 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	max-width: 880px;
+	margin: 25px;
+	
+	@media ( max-width: 500px){
+		display: flex;
+		flex-direction: column;
+
+	}
+	body{
+		border: .25rem dashed blue;
+		padding: 10px;
+		height: auto;
+		@media ( max-width: 500px){
+			border: none;
+		}
+	}
+	h1, h2 {
+		.center;
+		font-family: @fontheader;
+		font-size: 5rem;
+	}
+	h1{
+		color: #b7950b;
+	}
+	h2{
+		color: #95a5a6;
+	}
+	nav{
+		.customflex;
+		margin: 25px;
+		@media ( max-width: 500px){
+			justify-content: space-between;
+		}
+	}
+	nav a{
+		color: orange;
+		font-size: 1.5rem;
```

Comment:
Nice job using hover. However, it could've been nested for DRYer code like so:
```
nav a {
    color: orange;
    font-size: 1.5rem;

    &:hover {
       color: aqua;
   }
}
```